src/plugins/plugin.h

```diff
@@ -283,6 +283,10 @@ struct janus_plugin {
 	 * @param[out] error An integer that may contain information about any error */
 	void (* const create_session)(janus_plugin_session *handle, int *error);
 	/*! \brief Method to handle an incoming message/request from a peer
+	 * @note The Janus core increases the references to both the message and jsep
+	 * json_t objects. This means that you'll have to decrease your own
+	 * reference yourself with a \c json_decref when you're done with it.
+	 * You'll also have to free transaction with \c g_free.
```

Comment:
This is missing the `\c` before `transaction` I mentioned in my previous review.